tensorflow/lite/toco/graph_transformations/tests/identify_lstm_test.cc

```diff
@@ -0,0 +1,172 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
```

Comment:
This should be 2019 :)

Reply:
@jianlijianli , thanks for the review i have updated your comments along with removing of the logging dependency from the BUILD file, kindly check and approve.

Sorry for the inconvenience caused.

Regards
Amit